src/api/process.ts

```diff
@@ -33,48 +33,65 @@ export function startBuildProcess(
   image: string
 ): Observable<ProcessOutput> {
   return new Observable(observer => {
-    const name = 'abstruse_' + proc.build_id + '_' + proc.job_id;
-    const vars = proc.commands.filter(cmd => cmd.command.startsWith('export'))
-      .map(cmd => cmd.command.replace('export', '-e'))
-      .reduce((acc, curr) => {
-        return acc.concat(curr.split(' '));
-      }, [])
-      .concat(proc.env.reduce((acc, curr) => acc.concat(['-e', curr]), []));
-    proc.commands = proc.commands.filter(cmd => !cmd.command.startsWith('export'));
+    getRepositoryByBuildId(proc.build_id).then(repo => {
```

Comment:
why did you add new method for getting repository if `getBuild(proc.build).then(build => build.repository` works just fine?